integration_test/ownership/manager_test.exs

```diff
@@ -575,15 +575,15 @@ defmodule ManagerTest do
       {:disconnect, RuntimeError.exception("oops"), TestConnection, :state}
     end
 
-    assert capture_log(fn ->
+    capture_log(fn ->
```

Comment:
Perhaps we should keep the assertion and update it just in case? Especially as you said they report errors different?

Reply:
@josevalim yeah, I had to add a manual log again. I’m not sure why gen_statem is not logging anything when stopping with reason other than normal/shutdown 😵‍💫